src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -311,6 +314,14 @@ public Task CopyToAsync(Stream stream)
             return CopyToAsync(stream, null);
         }
 
+#if NET46
+        // Workaround for HttpWebRequest synchronous resubmit
```

Comment:
Please add more information here. What is the sync resubmit issue? (link or code ref)